src/main/java/xbdd/webapp/util/DatabaseUtilities.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2015 Orion Health (Orchestral Development Ltd)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package xbdd.webapp.util;
+
+import com.mongodb.BasicDBList;
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+
+public final class DatabaseUtilities {
+	private DatabaseUtilities() {

```

Comment:
Yeah I was talking to someone at work about it and apparently it is relatively standard practice to prevent instantiations.  I think it can stay around :+1: